.github/workflows/ruby.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0']
```

Comment:
I think Ruby 2.6 is EOL and 2.7 is almost EOL?
What do you think of `['2.7', '3.0', '3.1', '3.2']`?

Reply:
Yeah, that's a reasonable change. This just came from the default Workflow setup from Github, so I'm not glued to it.

Reply:
Updated!